sensor/kubernetes/listener/resources/route.go

```diff
@@ -42,6 +44,11 @@ func (r *routeDispatcher) ProcessEvent(obj, _ interface{}, action central.Resour
 	if existingService == nil {
 		return nil
 	}
+	if env.ResyncDisabled.BooleanSetting() {
+		event := component.NewEvent()
+		event.AddDeploymentReference(resolver.ResolveDeploymentLabels(existingService.GetNamespace(), existingService.selector), central.ResourceAction_UPDATE_RESOURCE, false)
+		return event
```

Comment:
Could you just add a comment here explaining that when the re-sync is disabled, we don't want to call `UpdateExposuresForMatchingDeployments` here and instead reprocess all the matching deployments. And also that we only send the event with the deployment reference because services **don't send any resource events** to central. It looks a bit different from the other handlers because of that. 

When I first read the code I forgot that services were not sent, and it would be good to have the context. Probably the comment can be removed once the old path is also removed.

Reply:
Done!